index.html

```diff
@@ -0,0 +1,366 @@
+    style="width: 1024px; height: 400px; border:0; transform: scale(1); overflow:hidden;"
+    sandbox="allow-scripts allow-same-origin">
+            </iframe>
+            <h2 id="badge">Badge</h2>
+            <p>
+                Avatar component is typically used to display circular user profile pictures. This component will allow you to dynamically size. You can use sm md lg elg to change the size of the avatar.
+            </p>
+            <div style="height: 25px;"></div>
+            <span class="badge">
+                <i class="material-icons">shopping_cart</i>
+                <span>3</span>
+            </span>
+            <span style="margin-right: 50px;"></span>
+            <span class="badge">
+                <i class="material-icons">favorite_border</i>
+                <span>10</span>
+            </span>
+            <span style="margin-right: 50px;"></span>
+            <span class="badge">
+                <i class="material-icons md-48">shopping_cart</i>
+                <span>3</span>
+            </span>
+            <span style="margin-right: 50px;"></span>
+            <span class="badge">
+                <i class="material-icons md-48">favorite_border</i>
+                <span>10</span>
+            </span>
+            <div style="height: 25px;"></div>
+            <div style="text-align: center;">
+                <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%253Cspan%2520class%253D%2522badge%2522%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522material-icons%2522%253Eshopping_cart%253C%252Fi%253E%250A%2520%2520%2520%2520%253Cspan%253E3%253C%252Fspan%253E%250A%253C%252Fspan%253E%250A%250A%253Cspan%2520class%253D%2522badge%2522%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522material-icons%2522%253Efavorite_border%253C%252Fi%253E%250A%2520%2520%2520%2520%253Cspan%253E10%253C%252Fspan%253E%250A%253C%252Fspan%253E%250A%250A%253Cspan%2520class%253D%2522badge%2522%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522material-icons%2520md-48%2522%253Eshopping_cart%253C%252Fi%253E%250A%2520%2520%2520%2520%253Cspan%253E3%253C%252Fspan%253E%250A%253C%252Fspan%253E%250A%250A%253Cspan%2520class%253D%2522badge%2522%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522material-icons%2520md-48%2522%253Efavorite_border%253C%252Fi%253E%250A%2520%2520%2520%2520%253Cspan%253E10%253C%252Fspan%253E%250A%253C%252Fspan%253E"
+                style="width: 614px; height: 536px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </div>
+            <div style="font-size: 1.2em; font-weight: 600;">Badges for avatar</div>
```

Comment:
**This looks great to me.**
Try to avoid using  style tags inside HTML

Thanks!

Reply:
Cool. Will Remember while refactoring the code. Thanks